pkgs/development/compilers/gforth/default.nix

```diff
@@ -6,15 +6,15 @@ let
   swig = callPackage ./swig.nix { };
   bootForth = callPackage ./boot-forth.nix { };
   lispDir = "${placeholder "out"}/share/emacs/site-lisp";
-in stdenv.mkDerivation rec {
+in stdenv.mkDerivation (finalAttrs: {
 
   pname = "gforth";
-  version = "0.7.9_20230518";
+  version = "0.7.9_20240229";
 
   src = fetchFromGitHub {
     owner = "forthy42";
     repo = "gforth";
-    rev = version;
+    rev = finalAttrs.version;
     hash = "sha256-rXtmmENBt9RMdLPq8GDyndh4+CYnCmz6NYpe3kH5OwU=";
```

Comment:
You must update this hash

Reply:
yes, i know, that's why this is a draft.  the package doesn't build with the new version.